src/main/java/org/lobid/resources/run/AlmaMarcXmlFix2lobidJsonEs.java

```diff
@@ -128,9 +128,9 @@ public void run() {
                 fixVariables.put("rvk.tsv", "../../../../../../lookup-tables/data/rvk.tsv");
                 fixVariables.put("lobidOrganisationsMapping.tsv", "./maps/lobidOrganisationsMapping.tsv");
                 fixVariables.put("hbzowner2sigel.tsv", "./maps/hbzowner2sigel.tsv");
-                fixVariables.put("rpb2.ttl", "../../../../../../vocabs/rpb/rpb2.ttl");
-                fixVariables.put("rpb-spatial.ttl", "../../../../../../vocabs/rpb/rpb-spatial.ttl");
-                fixVariables.put("rpb.ttl", "../../../../../../vocabs/rpb/rpb.ttl");
+                fixVariables.put("rpb2.ttl", "../../../../../../lbz-vocabs/rpb/rpb2.ttl");
```

Comment:
This file will finally most prbably be renamed to `lbz-notationen.ttl`, see https://github.com/hbz/lbz-vocabs/pull/6/commits/38b3e89a0473090aa52154dd86644fad43a8f818

Reply:
This means, we should wait with mergin this until https://github.com/hbz/lbz-vocabs/pull/6 is merged and adjust the file name if necessary.

Reply:
I have made a separate PR for changing the file name at https://github.com/hbz/lbz-vocabs/pull/7 so that we can move on with this PR. @TobiasNx will review over there.